fusedrug/data/tokenizer/injectortokenizer/injector_tokenizer.py

```diff
@@ -0,0 +1,207 @@
+from fusedrug.data.tokenizer.modulartokenizer.modular_tokenizer import ModularTokenizer
+from typing import Optional, List, Tuple, Dict
+from tokenizers import Encoding
+import torch
+import re
+from fuse.utils import NDict
+
+
+class InjectorTokenizer(ModularTokenizer):
```

Comment:
originally I wanted to use this as a drop-in replacement for ModularTokenizer, but in order to avoid code duplication I ended up only storing static methods here.